anyway/models.py

```diff
@@ -876,6 +876,7 @@ class NewsFlash(Base):
     street2_hebrew = Column(Text(), nullable=True)
     non_urban_intersection_hebrew = Column(Text(), nullable=True)
     road_segment_name = Column(Text(), nullable=True)
+    road_segment_id = Column(Integer(), nullable=True)
```

Comment:
@ziv17 we need an alembic version file fir this modification

Reply:
This field is already in the table.